utils/common/src/main/java/org/apache/brooklyn/util/ssh/BashCommands.java

```diff
@@ -621,6 +621,7 @@ private static Object getFlag(Map<?,?> flags, String flagName, Object defaultVal
     public static String installJava(int version) {
         Preconditions.checkArgument(version == 6 || version == 7 || version == 8, "Supported Java versions are 6, 7, or 8");
         List<String> commands = new LinkedList<String>();
+        commands.add(ok(addOpenJDKPPK()));
```

Comment:
OK ignores if the command fails (so if the OS isn't Ubuntu for instance) so this should be fine. It's the pattern used elsewhere for similar things - see the [lines below](https://github.com/drigodwin/brooklyn-server/blob/b6c8566bbbc4766dd5e332a9c0db7f992f5f9ed4/utils/common/src/main/java/org/apache/brooklyn/util/ssh/BashCommands.java#L633).